pkg/app/rds_aurora_mysql.go

```diff
@@ -328,6 +354,20 @@ func (a *RDSAuroraMySQLDB) Uninstall(ctx context.Context) error {
 		return errors.Wrap(err, "Failed to create ec2 client.")
 	}
 
+	log.Info().Print("Deleting db subnet group.", field.M{"app": a.name})
+	_, err = rdsCli.DeleteDBSubnetGroup(ctx, a.dbSubnetGroup)
+	if err != nil {
+		// If the subnet group does not exist, ignore the error and return
+		if err, ok := err.(awserr.Error); ok {
+			switch err.Code() {
+			case awsrds.ErrCodeDBSubnetGroupNotFoundFault:
+				log.Info().Print("Subnet Group Does not exist: ErrCodeDBSubnetGroupNotFoundFault.", field.M{"app": a.name, "name": a.dbSubnetGroup})
```

Comment:
missing return?

Reply:
yeah, because we don't want to return in case its not found.